caddyhttp/rewrite/setup.go

```diff
@@ -90,10 +90,11 @@ func rewriteParse(c *caddy.Controller) ([]httpserver.HandlerConfig, error) {
 			if to == "" {
 				return nil, c.ArgErr()
 			}
-			if rule, err = NewComplexRule(base, pattern, to, ext, matcher); err != nil {
+			cRule, err := NewComplexRule(base, pattern, to, ext, matcher)
+			if err != nil {
 				return nil, err
 			}
-			rules = append(rules, rule)
+			rules = append(rules, *cRule)
```

Comment:
Is there a reason that NewComplexRule is still returning a pointer, if it's just as good without?